tools/fuzzy_proof_generator_tool/patterns.ts

```diff
@@ -0,0 +1,15 @@
+const patterns: string[] = [
```

Comment:
should some patterns end on a branch node too?

Reply:
I will add those. Please node though, that due to the solidity implementation of the MPP library, we can only end on *two* branch nodes.

Working examples:
* `bbb`
* `bebb`

Non-working examples:
* `eb`
* `bbeb`